lib/reporters/cli/index.js

```diff
@@ -147,7 +148,22 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
     emitter.on('beforeRequest', function (err, o) {
```

Comment:
isn't timestamp needed for other events like the script, console, etc?

Reply:
I implemented what the PR asked for, though adding timestamp for all CLI reported events does seem to have advantages (like checking out what part of the collection is slowing things down). 

It can be added but should I add it in the current flag or will it be okay to add it in the current flag. I'd like some input on this :)